src/controller/config.js

```diff
@@ -173,12 +211,19 @@ const getClickCoordinates = (event) => {
     bodyTagModal.show()
 }
 
+const populateEditChecklistBodyParts = (bodyParts) => {
+    for(let key in bodyParts){
```

Comment:
populate the body part map with the body parts from the checklist we are editing